cmake/modules/RootMacros.cmake

```diff
@@ -1086,6 +1236,7 @@ endfunction()
 #                                 LIBRARIES lib1 lib2          : linking flags such as dl, readline
 #                                 DEPENDENCIES lib1 lib2       : dependencies such as Core, MathCore
 #                                 BUILTINS builtin1 builtin2   : builtins like AFTERIMAGE
+#                                 INCLUDES inc1 inc2           : extra includes paths to build sources
```

Comment:
This should be fed in through `target_include_directories` from the outside.

Reply:
Removed